From c483863600425111728c4df8db417a43f28be167 Mon Sep 17 00:00:00 2001
From: Ookiineko <chiisaineko@protonmail.com>
Date: Wed, 13 Dec 2023 13:17:00 +0800
Subject: [PATCH] base: disable parse_mount_info

* this function is unused by magiskboot, and it uses some new compiler feature,
  unfriendly to old distros

Signed-off-by: Ookiineko <chiisaineko@protonmail.com>
---
 native/src/base/files.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/native/src/base/files.cpp b/native/src/base/files.cpp
index 050bda6..3d7477e 100644
--- a/native/src/base/files.cpp
+++ b/native/src/base/files.cpp
@@ -127,6 +127,7 @@ void parse_prop_file(const char *file, const function<bool(string_view, string_v
         parse_prop_file(fp.get(), fn);
 }
 
+#if 0
 std::vector<mount_info> parse_mount_info(const char *pid) {
     char buf[PATH_MAX] = {};
     ssprintf(buf, sizeof(buf), "/proc/%s/mountinfo", pid);
@@ -206,6 +207,7 @@ std::vector<mount_info> parse_mount_info(const char *pid) {
     });
     return result;
 }
+#endif
 
 sDIR make_dir(DIR *dp) {
     return sDIR(dp, [](DIR *dp){ return dp ? closedir(dp) : 1; });
-- 
2.42.0

